src/views/List.jsx

```diff
@@ -1,15 +1,44 @@
+import { useState } from 'react';
+
 import { ListItem } from '../components';
 
 export function List({ data }) {
+	const [searchTerm, setSearchTerm] = useState('');
+
+	const clearSearchTerm = () => {
+		setSearchTerm('');
+	};
+
 	return (
 		<>
 			<p>
 				Hello from the <code>/list</code> page!
 			</p>
+			<form onSubmit={(e) => e.preventDefault()}>
```

Comment:
Since the form logic isn't being used to link up to a `submit` button or save form data, this <form> wrapper might be unneccessary!

 If you guys wanted to keep the form wrapper and not have to worry about the `onSubmit` preventing Default as well, you could completely remove the `onSubmit` without changing any functionality too.